apps/demos/Demos/Chat/AIIntagration/test-code.js

```diff
@@ -0,0 +1,3 @@
+testUtils
+  .postponeUntilFound('.dx-chat')
+  .then(() => testUtils.findElements('.dx-chat').forEach((x) => { x.style.height = '700px'; }));
```

Comment:
What it's necessary for?

Reply:
It's not necessary for the chat to fit within a 1000 x 800 area in the screenshot.
Chat's dimensions are  height: 600, width: 520 and we can remove this file.

Reply:
[0f60119da81def3f3f4e8a295eaefda87674892c]